lib/src/animated_circular_chart.dart

```diff
@@ -218,7 +218,7 @@ class AnimatedCircularChartState extends State<AnimatedCircularChart>
   void _updateLabelPainter() {
     if (widget.holeLabel != null) {
       TextStyle _labelStyle =
-          widget.labelStyle ?? Theme.of(context).textTheme.body2;
+          widget.labelStyle ?? Theme.of(context).textTheme.bodyText1;
```

Comment:
was this text style change intentional?

Reply:
Yes! Good question though. The old value is the deprecated one that points to this new one.